node/_events.d.ts

```diff
@@ -0,0 +1,788 @@
+// deno-lint-ignore-file no-explicit-any
+
+// Forked from https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9b9cd671114a2a5178809798d8e7f4d8ca6c2671/types/node/events.d.ts
+
+export const captureRejectionSymbol: unique symbol;
+export const defaultMaxListeners: number;
+export const errorMonitor: unique symbol;
+
+/**
+ * Returns a copy of the array of listeners for the event named `eventName`.
+ *
+ * For `EventEmitter`s this behaves exactly the same as calling `.listeners` on
+ * the emitter.
+ *
+ * For `EventTarget`s this is the only way to get the event listeners for the
+ * event target. This is useful for debugging and diagnostic purposes.
+ *
+ * ```js
+ * const { getEventListeners, EventEmitter } = require('events');
+ *
+ * {
+ *   const ee = new EventEmitter();
+ *   const listener = () => console.log('Events are fun');
+ *   ee.on('foo', listener);
+ *   getEventListeners(ee, 'foo'); // [listener]
+ * }
+ * {
+ *   const et = new EventTarget();
+ *   const listener = () => console.log('Events are fun');
+ *   et.addEventListener('foo', listener);
+ *   getEventListeners(et, 'foo'); // [listener]
+ * }
+ * ```
+ * @since v15.2.0
+ */
+export function getEventListeners(
```

Comment:
If you see duplicated comment blocks it's because VSCode doesn't seem to apply those when you do something like `x: typeof type_with_comments` so I figured it would make sense even if it's kinda ugly